ansible/group_vars/all

```diff
@@ -83,28 +86,19 @@ controller:
   loglevel: "{{ controller_loglevel | default(whisk_loglevel) | default('INFO') }}"
   entitlement:
     spi: "{{ controller_entitlement_spi | default('') }}"
-  protocol: "{{ controllerProtocolForSetup }}"
+  protocol: "{{ controller_protocol | default('https') }}"
   ssl:
     cn: openwhisk-controllers
-    cert: "{{ controller_ca_cert | default('controller-openwhisk-server-cert.pem') }}"
-    key: "{{ controller_key | default('controller-openwhisk-server-key.pem') }}"
-    clientAuth: "{{ controller_client_auth | default('true') }}"
+    keyPrefix: "{{ __controller_ssl_keyPrefix }}"
     storeFlavor: PKCS12
+    clientAuth: "{{ controller_client_auth | default('true') }}"
+    cert: "controller-openwhisk-server-cert.pem"
```

Comment:
Should we use this var `__controller_ssl_keyPrefix` for both cert and key variables?